### PR TITLE
FullWarApi

### PR DESCRIPTION
**Pull request by doluk**
Extension coded similar to discordlinks to give selected persons access to the FullWarApi, which solves the problems around the war cache if war needs to be tracked.
You can find the documentation [here](https://fw-api.teamutils.com/docs). An account can be requested in my DMs.

Thanks to @MagicTheDev for implementing the extension for coc.py.

I suggest we merge it with a single commit!

### CODE REVIEW

**Comment by doluk** [comment]
on hold please, found a bug

**Comment by Kuchenmampfer** [comment]
Does someone already use this? Is the bug fixed?
This does not include docs right? I think long term we should add integrated docs like we have for the dc links extension. However, I personally would also be fine if we link https://fw-api.teamutils.com/docs somewhere in the docs as a short term solution for the initial merge.

**Comment by doluk** [comment]
@Kuchenmampfer @MagicTheDev uses it. If you want to test it, let me know. I can give you creds

**Comment by doluk** [comment]
It should be stable now, we just need to resolve the conflicts, where ever they come from

File: coc/ext/fullwarapi/__init__.py [hunk 1]
```diff
@@ -216,6 +216,6 @@ async def register_war(self, clan_tag: str, preparation_start: int = 0):
             Preparation time of the war
         """
         return await self._request("POST",
-                                   f"/war_result?clan_tag={correct_tag(clan_tag, '%23')}"
+                                   f"/register_war?clan_tag={correct_tag(clan_tag, '%23')}"
```
Comment (doluk) [inline coc/ext/fullwarapi/__init__.py:219]: register_war is the correct one

**Pull request closed without merging**